torcharrow/test/test_functional_cpu.py

```diff
@@ -51,7 +51,16 @@ def test_functional_dispatch(self):
         # Validate that invoking unknown UDFs errors nicely.
         with self.assertRaises(RuntimeError) as ex:
             assert functional.idontexist(str_col)
-        self.assertEqual(str(ex.exception), "Request for unknown Velox UDF: idontexist")
+        self.assertTrue(
+            str(ex.exception).startswith("Request for unknown Velox UDF: idontexist")
+        )
+
+        # Validate that invoking unknown UDFs with unsupported signatures errors nicely too.
+        with self.assertRaises(RuntimeError) as ex:
+            assert functional.firstx(str_col, 1)
+        self.assertTrue(
+            str(ex.exception).startswith("Velox UDF signature is not supported")
```

Comment:
nit: curious if we can also include the passed in signature, and maybe one expected signature in tests? ?  Something like 
```python
msg = str(ex.exception)

self.assertTrue(msg.startswith("Velox UDF signature is not supported: XXXXX"))
supported_sig = msg[msg.find("Supported signatures:"):]
self.assertTrue("XXXX" in supported_sig)
```